### ISSUE TITLE
Not able to read Nicolet/Nervus files from before 2012

### ISSUE THREAD

**Issue by janbrogger** [fieldtrip/fieldtrip#1357]
The code in pull request #8 https://github.com/robertoostenveld/fieldtrip/pull/8 has been tested and is not able to read our clinical EEG files from 2004 to 2012. There is a file format change. 

The issue appears to be that there is an index of some sort that is not present:
in fileio/private/read_nervus_header lines 43-47:
```
nrvHdr.indexIdx = fread(h,1,'uint32');
if (nrvHdr.indexIdx==0)
    fclose(h);
    ft_error('Unsupported old-style Nicolet file format (pre-ca. 2012)');
end    
```

**Comment by janbrogger** [comment]
Please assign this issue to me

**Comment by schoffelen** [comment]
Will do, but this has now been resolved with the latest PR (#1358), right?

**Comment by janbrogger** [comment]
> Will do, but this has now been resolved with the latest PR (#1358), right?

No, this issue is not fixed in PR #1358 . Files from before ca. 2012 cannot be read. I need to study the file format more. I have an old specification from 1996, and various bits of code from others.

**Comment by schoffelen** [comment]
Closing due to lack of activity. Feel free to reopen if the issue is prioritized and will be worked on
